pkg/sensors/tracing/lists.go

```diff
@@ -7,22 +7,26 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/cilium/tetragon/pkg/arch"
 	"github.com/cilium/tetragon/pkg/btf"
 	"github.com/cilium/tetragon/pkg/ftrace"
 	gt "github.com/cilium/tetragon/pkg/generictypes"
 	"github.com/cilium/tetragon/pkg/k8s/apis/cilium.io/v1alpha1"
-	"github.com/cilium/tetragon/pkg/syscallinfo"
 )
 
-func getList(name string, lists []v1alpha1.ListSpec) *v1alpha1.ListSpec {
+// isList checks if a value specifics a list, and if so it returns it (or nil if list does not exist)
+func isList(val string, lists []v1alpha1.ListSpec) (bool, *v1alpha1.ListSpec) {
```

Comment:
do we need to return bool? returning just list and callers checking it for nil should be enought, right?

Reply:
nah it's used in generic functions like getKprobeSymbols to recognize list, ok then ;-)